Nodejs/Product/Nodejs/Intellisense/IntellisenseController.cs

```diff
@@ -1,663 +1,772 @@
+            // on the text view.
+            //
+            // We trigger signature help when we receive a "(".  We update our current sig when 
+            // we receive a "," and we close sig help when we receive a ")".
+
+            if (!_incSearch.IsActive) {
+                switch (ch) {
+                    case '.':
+                    case ' ':
+                        if (NodejsPackage.Instance.LangPrefs.AutoListMembers) {
+                            TriggerCompletionSession(false);
+                        }
+                        break;
+                    case '/':
+                    case '\'':
+                    case '"':
+                        if (CompletionSource.ShouldTriggerRequireIntellisense(_textView.Caret.Position.BufferPosition, _classifier, true, true)) {
+                            TriggerCompletionSession(false);
+                        }
+                        break;
+                    case '(':
+                        if (CompletionSource.ShouldTriggerRequireIntellisense(_textView.Caret.Position.BufferPosition, _classifier, true)) {
+                            TriggerCompletionSession(false);
+                        } else if (NodejsPackage.Instance.LangPrefs.AutoListParams) {
+                            OpenParenStartSignatureSession();
+                        }
+                        break;
+                    case ')':
+                        if (_sigHelpSession != null) {
+                            _sigHelpSession.Dismiss();
+                            _sigHelpSession = null;
+                        }
+
+                        if (NodejsPackage.Instance.LangPrefs.AutoListParams) {
+                            // trigger help for outer call if there is one
+                            TriggerSignatureHelp();
+                        }
+                        break;
+                    case '=':
+                    case ',':
+                        if (_sigHelpSession == null) {
+                            if (NodejsPackage.Instance.LangPrefs.AutoListParams) {
+                                CommaStartSignatureSession();
+                            }
+                        } else {
+                            UpdateCurrentParameter();
+                        }
+                        break;
+                }
+            }
+        }
+
+        private bool Backspace() {
+            if (_sigHelpSession != null) {
+                if (_textView.Selection.IsActive && !_textView.Selection.IsEmpty) {
+                    // when deleting a selection don't do anything to pop up signature help again
+                    _sigHelpSession.Dismiss();
+                    return false;
+                }
+
+                SnapshotPoint? caretPoint = _textView.BufferGraph.MapDownToFirstMatch(
+                    _textView.Caret.Position.BufferPosition,
+                    PointTrackingMode.Positive,
+                    EditorExtensions.IsNodeJsContent,

```

Comment:
If you're going to make this change, you may as well remove the IsNodejsContent method in this file altogether since the two functions are exactly the same.